test/build/config/undefined/webpack.config.js

```diff
@@ -0,0 +1 @@
+module.exports = undefined;
```

Comment:
For test let's create just empty file, but keep this test too

Reply:
Okay, will add a new test case.